### ISSUE TITLE
NPE in com.appnexus.opensdk.MRAIDImplementation

### ISSUE THREAD

**Issue by miduch** [appnexus/mobile-sdk-android#66]
I'm trying appnexus sdk in jetpack compose/ lazycolumn and randomly getting following NPE thrown when scrolling fast

```
java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.ViewGroup$LayoutParams com.appnexus.opensdk.AdView.getLayoutParams()' on a null object reference
                                        at com.appnexus.opensdk.MRAIDImplementation$1.run(MRAIDImplementation.java:126)
                                        at android.os.Handler.handleCallback(Handler.java:938)
                                        at android.os.Handler.dispatchMessage(Handler.java:99)
                                        at android.os.Looper.loopOnce(Looper.java:201)
                                        at android.os.Looper.loop(Looper.java:288)
                                        at android.app.ActivityThread.main(ActivityThread.java:7829)
                                        at java.lang.reflect.Method.invoke(Native Method)
                                        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                                        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:982)
```

This handler should either be canceled if/when adview is destroyed or should check if owner and adview are not null in run()  
 
https://github.com/appnexus/mobile-sdk-android/blob/master/sdk/src/com/appnexus/opensdk/MRAIDImplementation.java#L126

Probably same issue in other places in sdk as well which are not discovered yet. 

Thanks!

**Comment by avohraa** [comment]
Hi @miduch, we will be addressing this in the next release.

**Comment by avohraa** [comment]
Hi @miduch, this issue has been addressed in SDK v7.17 (https://github.com/appnexus/mobile-sdk-android/releases/tag/7.17). Feel free to reopen if that issue arises again.
